cmake/modules/holohub_configure_deb.cmake

```diff
@@ -59,6 +59,7 @@ function(holohub_configure_deb)
   endif()
 
   # standard configurations
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/opt/nvidia/holoscan")
```

Comment:
Should default installation be to `/opt/nvidia/holoscan` or distinct `/opt/nvidia/holohub` or similar?

Reply:
I think at this stage probably fully subjective, and affects the question here also: https://github.com/nvidia-holoscan/holohub/pull/705#issuecomment-2685421095

If we think about holohub packaged operators/apps as "modules" to expand holoscan, installing it in `/opt/nvidia/holoscan` makes sense and makes ld and python import discovery easier by piggy backing on the HSDK paths.

Reply:
Fair point, if folder pollution doesn't seem like a concern then not a blocker. Agreed it's nice to not have to update ldconfig and PYTHONPATH for libraries. PATH update may be necessary regardless of resolution in https://github.com/nvidia-holoscan/holohub/pull/705#:~:text=question%20here%20also%3A-,%23705%20(comment),-If%20we%20think.